telemetry/specs/INT.mdk

```diff
@@ -384,12 +384,32 @@ ignore the designated bit position.)
 * TCP/UDP destination Port field: A port number in layer 4 will be reserved to
 indicate the existence of INT after TCP/UDP. INT source changes the destination
 Port, and sink must restore the original port number.
+* New Probe Marker fields: arbitrary 32-bit values are inserted after IP TCP/UDP header
+to indicate the existence of INT after TCP/UDP. These fileds should be interpreted
+as unsigned integer values and are initialized to a configured value. 
 
-The decision to use DSCP or destination port field and the reserved values are
+INT probe marker for TCP/UDP:
+
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                         Probe Marker (1)                      |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                         Probe Marker (2)                      |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+
+The decision to use DSCP, destination port field or probe marker, and 
+the reserved or configured values are
 per domain decisions that all INT capable devices should follow. (Note: in
 general, encoding into DSCP field will be less intrusive compared to changing
 the layer 4 port field. The latter may alter ECMP behavior and can complicate
-ACL and network debugging.)
+ACL and network debugging. If DSCP field can not be reseved for INT, and 
+retaining ECMP behavior is desired, adding probe marker fields could be an
+option. With arbitary values being inserted after TCP/UDP header, the chance
+to conflict with user traffic in a data center is low, but can not be completely 
+eliminated.)
```

Comment:
Do you want to mention the optional check on TCP/UDP port numbers?

Reply:
can not -> cannot

Reply:
Added the port check and also corrected to "cannot"